docs/Control.Monad.Aff.md

```diff
@@ -159,6 +159,16 @@ will not block on the result of the computation.
 Returns a canceler that can be used to attempt cancellation of the
 forked computation.
 
+#### `forkAll`
+
+``` purescript
+forkAll :: forall f e a. (Foldable f) => f (Aff e a) -> Aff e Unit
+```
+
+Forks many asynchronous computation at once, ignoring the results.
```

Comment:
It would be good to add a guarantee that all computations will be forked synchronously.